Celeste.Mod.mm/Mod/Helpers/TypeHelper.cs

```diff
@@ -0,0 +1,316 @@
+﻿using Celeste.Mod.Core;
+using Celeste.Mod.Entities;
+using Monocle;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Celeste.Mod.Helpers {
+    public static class TypeHelper {
+
+        public static Dictionary<string, Type> typeFullNameToTypeCache;
```

Comment:
This field is never set and is always null, makes `typefinder` crash when provided a c# type